### PR TITLE
MAT-3265 Set Measurement Period seconds values to zero

### PR DESCRIPTION
**Pull request by jkotanchik-SB**
Since Bonnie cannot create patient dateTime data with seconds, always set the measurement period seconds (milli) values to zero to ensure compatible comparisons.

Pull requests into cqm-parsers require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: [MAT-3265](https://jira.cms.gov/browse/MAT-3265)
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

**Review by serhii-ilin — APPROVED** [review]

File: lib/util/bundle_utils.rb [hunk 1]
```diff
@@ -29,10 +29,10 @@ def self.get_measurement_period(fhir_measure)
         end
       mp[:end] =
         if fhir_measure.effectivePeriod.end&.value
-          fhir_measure.effectivePeriod.end.value << "T23:59:59"
+          fhir_measure.effectivePeriod.end.value << "T23:59:00"
```
Comment (serhii-ilin) [inline lib/util/bundle_utils.rb:32]: Do we need to handle HH:MM:SS from MAT?
It can potentially have all parts of DateTime value

Comment (jkotanchik-SB) [reply lib/util/bundle_utils.rb:32]: Currently, MAT does not record time for Measurement period, only Date, and I'm not aware of any upcoming change that will add it. Good thing to try and keep in mind, though. Nice catch!

Comment (serhii-ilin) [reply lib/util/bundle_utils.rb:32]: Kathy mentioned DateTime in MP, Not sure if it affects effectivePeriod

**Review by adongare — APPROVED** [review]
